robot-server/robot_server/commands/router.py

```diff
@@ -0,0 +1,96 @@
+"""Router for top-level /commands endpoints."""
+from anyio import move_on_after
+from typing_extensions import Final, Literal
+
+from fastapi import APIRouter, Depends, Query, status
+
+from opentrons.protocol_engine import Command, CommandCreate
+
+from robot_server.errors import ErrorDetails, ErrorBody
+from robot_server.runs import EngineStore, EngineConflictError, get_engine_store
+from robot_server.service.json_api import RequestModel, SimpleBody, PydanticResponse
+
+_DEFAULT_COMMAND_WAIT_MS: Final = 30_000
+
+commands_router = APIRouter()
+
+
+class RunActive(ErrorDetails):
+    """An error returned if there is a run active.
+
+    If there is a run active, you cannot issue stateless commands.
+    """
+
+    id: Literal["RunActive"] = "RunActive"
+    title: str = "Run Active"
+    detail: str = (
+        "There is an active run. Close the current run"
+        " to issue commands via POST /commands."
+    )
+
+
+@commands_router.post(
+    path="/commands",
+    summary="Add a command to be executed.",
+    description=(
+        "Run a single command on the robot. This endpoint is meant for"
+        " simple, stateless control of the robot. For complex control,"
+        " create a run with ``POST /runs`` and issue commands on that run."
```

Comment:
Pedantically, I think it'd be more conventional to use single backticks here, since this is a Markdown string, not a Sphinx/reStructuredText string.

Double backticks *will* render correctly because Markdown allows an arbitrary number of backticks, but it's not conventional.

```suggestion
        " create a run with `POST /runs` and issue commands on that run."
```